02-express-tutorial/practiceExpress.js

```diff
@@ -8,5 +8,7 @@ app.get('/sample', (req, res) => {
   res.status(200).send('This is working.');
 });
 
+app.use(consoleLog);
```

Comment:
You may already have noticed this since this is just the practice file; but just wanted to note that currently this will fail since the variable `consoleLog` is undefined.